src/main/java/org/openpnp/model/Part.java

```diff
@@ -58,12 +60,13 @@ public void configurationLoaded(Configuration configuration)
                     throws Exception {
                 if (getPackage() == null) {
                     setPackage(configuration.getPackage(packageId));
+//                    setLibrary(configuration.getLibrary(libraryId));
                 }
             }
 		});
 	}
 	
-	@SuppressWarnings("unused")
+//	@SuppressWarnings("unused")
```

Comment:
Why is this commented out? It is there to avoid warnings about unused functions.